### DISCUSSION TITLE
Uppercase Wildcards

### DISCUSSION THREAD

**Discussion by Justman100** [WeblateOrg/weblate#11274] [Q&A] [answered]
Hi, if one uses * as a wildcard in the _file mask_, only the language code is written in lower case! Can one change this?

**Comment by nijel** [comment]
There are several naming styles available for translation files created by Weblate: https://docs.weblate.org/en/latest/admin/projects.html#component-language-code-style

**Reply by Justman100** [reply]
But not said, what i need! I need like this _FR

**Reply by nijel** [reply, marked as answer]
I didn't notice the post title, that was the answer to the question “Can one change this?”.

There is currently no upper-cased option, you are welcome to contribute that. You can also create translation files outside Weblate, it will parse upper-cased names just fine.

**Reply by Justman100** [reply]
@nijel

Could this be added?

**Reply by nijel** [reply]
Yes, it can be added, pull requests are welcome! #9377 is an example how new language code style for new files can be added.

**Reply by Justman100** [reply]
> Yes, it can be added, pull requests are welcome! #9377 is an example how new language code style for new files can be added.

I was actually talking about you maintainers

**Reply by nijel** [reply]
The good starting point would be to file a [feature request](https://github.com/WeblateOrg/weblate/issues/new?assignees=&labels=&projects=&template=feature_request.yml) describing what exactly you want. Still, there are hundredths of other issues to address, so contributing this yourself would be definitely the fastest way to get it done.
